scenarioo-server/src/main/java/org/scenarioo/rest/application/ScenariooWebApplication.java

```diff
@@ -58,43 +59,19 @@ public void contextInitialized(final ServletContextEvent servletContextEvent) {
 	}
 
 	private void loadConfiguration(final ServletContextEvent servletContextEvent) {
-		LOGGER.info("  Loading configuration ...");
+		
 
-		final String configurationDirectory = configureConfigurationDirectoryFromServerContext(servletContextEvent);
-		final String configurationFilename = configureConfigurationFilenameFromServerContext(servletContextEvent);
+		final String configurationDirectoryPath = documentationPathLogic.getDocumentationPath(servletContextEvent);
+		LOGGER.info("  Configured scenarioo data directory: " + configurationDirectoryPath);
```

Comment:
@mi-we habe hier das Logging noch ein wenig verbessert (früher und anderer Text).